docs/platform/howto/use-aws-privatelinks.rst

```diff
@@ -0,0 +1,205 @@
+Using AWS PrivateLink with Aiven services
+=========================================
+
+AWS `PrivateLink <https://aws.amazon.com/privatelink/>`__ brings Aiven
+services to the selected virtual private cloud (VPC) in your AWS
+account. In a traditional setup that uses `VPC
+peering <https://help.aiven.io/en/articles/778836-using-vpc-peering>`__
+, traffic is routed through an AWS VPC peering connection to your Aiven
+services. With PrivateLink, you can create a VPC endpoint to your own
+VPC and access an Aiven service from that. The VPC endpoint creates
+network interfaces (NIC) to the sub-nets and availability zones that you
+choose and receives the private IP addresses that belong to the IP range
+of your VPC. The VPC endpoint is routed to your Aiven service located in
+one of Aiven's AWS accounts.
+
+You can enable PrivateLink for Aiven services located in project VPCs.
+Before you can set up AWS PrivateLink, `create a
+VPC <https://help.aiven.io/en/articles/778836>`__ and launch the
+services that you want to connect to that VPC. As there is no network
+routing between the VPCs, you can use any private IP range for the VPC,
+unless you also want to connect to the project VPC using VPC peering
+connections. This means that overlaps in the IP range are not an issue.
+
+You can use either the `Aiven web console <https://console.aiven.io>`__
+or the `Aiven CLI <https://github.com/aiven/aiven-client>`__ to set up
+AWS PrivateLink. You also need the AWS CLI to create a VPC endpoint.
+
+**Note:** Aiven for Apache Cassandra® and Aiven for M3 services do not
```

Comment:
Since this might be an old Help article, could we pls verify that this **note** is still valid?

Reply:
I have verified that the note is still valid, I created Cassandra and M3 serviced in a project AWS VPC and the option to create private links is not available.